main.js

```diff
@@ -0,0 +1,66 @@
+var request = require('request');
+var readline = require('readline-sync');
+
+
+var gitHubOrgName = readline.question("Type here GitHub organization name: ");
+
+var optionsObj = {
+    uri: '',
+    headers: {'User-Agent': 'Request-Promise' },
+    json: true
+};
+
+Object.defineProperty(optionsObj, 'url', {
+    set: function(value) {
+        this.url = value;
+    }
+});
+
+optionsObj.uri = 'https://api.github.com/orgs/' + gitHubOrgName + '/repos';
+
+function getRepos(options) {
+    return new Promise(function(resolve, reject) {
```

Comment:
You can wrap request function once and just use it instead of wrapping each time writing the same code:
```javascript
const requestPromisified = (options) => {
    return new Promise((resolve, reject) => {
        request(options, function(error, response, body) {
            if (error) {
                return reject(error);
            }
            resolve(body);
        });
    });
};
```